src/styles/_variables.scss

```diff
@@ -1,4 +1,5 @@
 @import '_sky-variables.scss';
+@import '_font-awesome-variables';
```

Comment:
so we can do that, though their variables include an _nav_   in the variable names for each icon.    eg.   $fa_var_cogs    $fa-var-angle-double-right